### DISCUSSION TITLE
Package idea: ggrepel but with manual adjustments

### DISCUSSION THREAD

**Discussion by brunomioto** [ggplot2-extenders/ggplot-extension-club#5] [Ideas]
It would be awesome if, after the ggrepel computations, we could open an interactive panel and make minor manual adjustments, like a drag and drop way to fix overwritten labels.

**Comment by teunbrand** [comment]
R graphics interactivity is generally not great, but I think {esquisse} is mostly fits the drag-and-drop bill of interacting with plots. I'm not too familiar with it, but taking that as a starting point should be more convenient than starting from scratch. As for ggrepel computations, these are updated every time the plot is re-drawn like changing the window size. This makes the coordinates of the label finnicky to use in a static fashion.

**Comment by EvaMaeRey** [comment]
@brunomioto maybe we are looking at related problems here: https://github.com/teunbrand/ggplot-extension-club/discussions/8  The proposal there is about adding information about a specific point.  I think you might be more interested in a blanket of callouts and minor adjustments of those?   I love to look at charts that add detail with call outs, but rarely add them, cause it seems like a tedious thing to do.  But I might not be aware of other developments that make this easier...  I put ideas in a repo and tried using with more examples here: https://github.com/EvaMaeRey/ggcallout

**Comment by EvaMaeRey** [comment]
Here is some more detailed information about ggpp's capabilities at manual adjustment+ggrepel, that might be getting at your proposal.  https://www.r4photobiology.info/galleries/nudge-and-repel.html  
![image](https://github.com/teunbrand/ggplot-extension-club/assets/10491910/84d7fd30-a417-45c5-8ef1-d8f9c4033a24)
